README.md

```diff
@@ -47,9 +47,11 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
    echo "---" >> cmd/clusterctl/examples/google/out/provider-components.yaml
    kustomize build vendor/sigs.k8s.io/cluster-api/config/default/ >> cmd/clusterctl/examples/google/out/provider-components.yaml
    ```
-1. Create a cluster:
+2. Create a cluster, ensuring that the locally-running clusterctl process has access to GCP apis:
```

Comment:
nit: Markdown will automatically create the correct numbers. It's convenient to just put `1.` on each line if you later want to move steps around or insert steps, because then a human doesn't need to re-number each line.